tmk_core/protocol/chibios/main.c

```diff
@@ -32,6 +32,11 @@
 #include "sendchar.h"
 #include "debug.h"
 #include "printf.h"
+#include "rgblight_reconfig.h"
+
+#if (defined(RGB_MIDI) | defined(RGBLIGHT_ANIMATIONS)) & defined(RGBLIGHT_ENABLE)
```

Comment:
I suspect this could just as well be:
```suggestion
#if (defined(RGB_MIDI) || defined(RGBLIGHT_ANIMATIONS)) && defined(RGBLIGHT_ENABLE)
```
and might make more sense, from a readability standpoint.

Reply:
damm another copy paste error, i need to do a pass for the other platforms.